internal/kafka/command_topic.go

```diff
@@ -142,6 +145,40 @@ func (c *command) provisioningClusterCheck(lkc string) error {
 	return nil
 }
 
+func (c *command) prepareAnonymousContext(cmd *cobra.Command) error {
+	kafkaBootstrap, err := cmd.Flags().GetString("kafka-bootstrap")
+	if err != nil {
+		return err
+	}
+
+	platform := &config.Platform{
+		Server: kafkaBootstrap,
+		Name:   strings.TrimPrefix(kafkaBootstrap, "https://"),
+	}
+
+	kafkaClusterCfg := &config.KafkaClusterConfig{
+		ID:        "anonymous-id",
+		Name:      "anonymous-cluster",
+		Bootstrap: kafkaBootstrap,
+		APIKeys:   map[string]*config.APIKeyPair{},
```

Comment:
Our code should be able to safely handle the case where this is `nil`, IMO.

Reply:
no it gave me a npe later

Reply:
Can we prevent it from panicking later?

Reply:
it still needs to be initialized. the other option is to add a `cluster.APIKeys = map[]{}` later